packages/fxa-payments-server/src/routes/ProductV2/SubscriptionCreate/index.stories.tsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
```

Comment:
These are all-new stories that I'm starting to build up to exercise the modes of the new payment flow. Still much in progress.